html-skeleton/index.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Skeletons</title>
+    <body>Hello, HTML!</body>
+  </head>
+</html>
```

Comment:
So you will learn about terms like children and siblings later down the line.

But your body tag should not be a child of the head element.

The `<head>` and `<body>` tags should be siblings.

Please slack me if you want clarification on that.